app/forms/ctc/consent_form.rb

```diff
@@ -38,7 +46,12 @@ def save
           source: @intake.source,
           type: @intake.type
       )
-      client = Client.create!(intake_attributes: intake_attributes, tax_returns_attributes: [tax_return_attributes])
+      efile_attrs = Client::EfileSecurityInformation.new(attributes_for(:efile_security_information))
+      client = Client.create!(
+        intake_attributes: intake_attributes,
+        tax_returns_attributes: [tax_return_attributes],
+        efile_security_information: efile_attrs
```

Comment:
i suggest using the same pattern we use for intake_attributes:

add `accepts_nested_attribtues_for :efile_security_information` to the Client model, then change the implementaiton here to: `efile_security_information: attributes_for(:efile_security_information)`